packages/fluentui/react-northstar/src/themes/teams/components/SplitButton/splitButtonStyles.ts

```diff
@@ -49,8 +49,8 @@ const splitButtonStyles: ComponentSlotStylesPrepared<SplitButtonStylesProps, Spl
     return {
       borderRadius: v.borderRadius,
       position: 'relative',
-      display: 'inline-block',
-
+      display: 'flex',
+      flexWrap: 'nowrap',
```

Comment:
I am thinking that this may deserves visual test for catching regression.

Reply:
Added